### ISSUE TITLE
Text not vertically centered in Header/searchBar

### ISSUE THREAD

**Issue by tokland** [GeekyAnts/NativeBase#1028]
Android 6.0 (Wiko Fever). From `package.json`:

```
    "dependencies": {
        "native-base": "2.2.1",
        "react": "16.0.0-alpha.12",
        "react-native": "0.46.1"
    }
```
I copied the example from the docs:

```
import { AppRegistry } from 'react-native';
import React, { Component } from 'react';
import { Container, Header, Item, Input, Icon, Button, Text } from 'native-base';

export default class SearchBarExample extends Component {
  render() {
    return (
      <Container>
        <Header searchBar rounded>
          <Item>
            <Icon name="ios-search" />
            <Input placeholder="Search" />
            <Icon name="ios-people" />
          </Item>
          <Button transparent>
            <Text>Search</Text>
          </Button>
        </Header>
      </Container>
    );
  }
}

AppRegistry.registerComponent('Playground', () => SearchBarExample);
```
I see the bar and the icons, but the placeholder -and also the text, if I write some- it's cut off:

![screenshot_20170710-182101](https://user-images.githubusercontent.com/24643/28028606-5ae49bd4-659d-11e7-82bd-da046639135e.png)

**Comment by faogustavo** [comment]
Same issue here, any solution?

**Comment by zcxzmzy** [comment]
Same issue on android,  any solution?

**Comment by faogustavo** [comment]
Mine was a little bit down the center. I applied the `marginTop: -4` and it went to correct position :)

**Comment by shivrajkumar** [comment]
@faogustavo @zcxzmzy Seems to be working fine in the latest version.

**Comment by raldred** [comment]
@shivrajkumar Hey, this bug is not fully resolved in 2.3.2. 

<img width="291" alt="screenshot 2017-10-07 19 13 53 1" src="https://user-images.githubusercontent.com/37096/31310668-12ecaeb0-ab94-11e7-9a0e-4a7061db06c9.png">

**Comment by SupriyaKalghatgi** [comment]
Fixed with NativeBase [2.3.5](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.5)
